src/bundle/Resources/config/services.yml

```diff
@@ -21,6 +21,10 @@ imports:
     - { resource: services/permissions.yml }
     - { resource: services/forms.yml }
 
+parameters:
+    # default content type icon
+    ezsystems.ezplatform_admin_ui.default_content_type_thumbnail: '/bundles/ezplatformadminui/img/ez-icons.svg#file'
```

Comment:
It was specified in the task (https://jira.ez.no/browse/EZP-29998) description:
> In case of a custom content type without defined custom icons, then the file icon from admin-ui will be used.